### ISSUE TITLE
myloader issue when importing tables with dash

### ISSUE THREAD

**Issue by kfi** [xelabs/go-mydumper#14]
> 2019/03/07 14:47:51.787967 loader.go:87:    	 [INFO] 	working.table[mydb.m2epro_amazon_listing_product_backup_2015-05-11]
>  2019/03/07 14:47:51.788896 loader.go:102:    	 [INFO] 	drop(overwrite.is.true).table[mydb.m2epro_amazon_listing_product_backup_2015-05-11]
> panic: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '-05-11' at line 1 (errno 1064) (sqlstate 42000)

**Comment by BohuTANG** [comment]
Hi,

My test on MySQL(5.7) works well.
1. table info
```
mysql> show create table mydb.`m2epro_amazon_listing_product_backup_2015-05-11`\G;
*************************** 1. row ***************************
       Table: m2epro_amazon_listing_product_backup_2015-05-11
Create Table: CREATE TABLE `m2epro_amazon_listing_product_backup_2015-05-11` (
  `a` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8
```

2.  dumper
```
./bin/mydumper -h 192.168.0.253 -P 3306 -u mock -p mock -db mydb  -o mydb.sql
 2019/03/11 10:14:32.563060 dumper.go:35:      [INFO]  dumping.database[mydb].schema...
 2019/03/11 10:14:32.564000 dumper.go:45:      [INFO]  dumping.table[mydb.m2epro_amazon_listing_product_backup_2015-05-11].schema...
 2019/03/11 10:14:32.564039 dumper.go:171:      [INFO]  dumping.table[mydb.m2epro_amazon_listing_product_backup_2015-05-11].datas.thread[1]...
 2019/03/11 10:14:32.564447 dumper.go:123:      [INFO]  dumping.table[mydb.m2epro_amazon_listing_product_backup_2015-05-11].done.allrows[1].allbytes[0MB].thread[1]...
 2019/03/11 10:14:32.564471 dumper.go:173:      [INFO]  dumping.table[mydb.m2epro_amazon_listing_product_backup_2015-05-11].datas.thread[1].done...
 2019/03/11 10:14:32.564501 dumper.go:191:      [INFO]  dumping.all.done.cost[0.00sec].allrows[1].allbytes[3].rate[0.00MB/s]
```

3. dump info
```
ls mydb.sql/
metadata  mydb-schema-create.sql  mydb.m2epro_amazon_listing_product_backup_2015-05-11-schema.sql  mydb.m2epro_amazon_listing_product_backup_2015-05-11.00001.sql
```

4. myloader
```
./bin/myloader -h 192.168.0.245  -P 3306 -u mock -p mock -d mydb.sql
 2019/03/11 10:15:45.108958 loader.go:76:      [INFO]  restoring.database[mydb]
 2019/03/11 10:15:45.109101 loader.go:87:      [INFO]  working.table[mydb.m2epro_amazon_listing_product_backup_2015-05-11]
 2019/03/11 10:15:45.113916 loader.go:111:      [INFO]  restoring.schema[mydb.m2epro_amazon_listing_product_backup_2015-05-11]
 2019/03/11 10:15:45.114109 loader.go:127:      [INFO]  restoring.tables[m2epro_amazon_listing_product_backup_2015-05-11].parts[00001].thread[2]
 2019/03/11 10:15:45.116282 loader.go:145:      [INFO]  restoring.tables[m2epro_amazon_listing_product_backup_2015-05-11].parts[00001].thread[2].done...
 2019/03/11 10:15:45.116339 loader.go:202:      [INFO]  restoring.all.done.cost[0.00sec].allbytes[0.00MB].rate[0.00MB/s]
```

can you give the table info?

**Comment by BohuTANG** [comment]
Hi @kfi 

Thanks for your reporting.
This bug fixed cd048b97be027433533f1e20401d90c8a52a6e65

Please git pull master again and rebuild.

**Comment by kfi** [comment]
works great, thank you
